packages/quip-apps-api/src/user.ts

```diff
@@ -5,6 +5,7 @@ export default class User {
     idValue: string = "mock-user";
     nameValue: string = "Mock User";
     languageValue: string = "en";
+    companyIdValue: string = "HWWAcAP8yS4";
```

Comment:
These are intentionally readable (so that consumers will be able to see that this clearly isn't real data), we should name this `mock-company` or something